src/ucp/wireup/wireup.c

```diff
@@ -762,50 +742,106 @@ ucp_wireup_connect_lane(ucp_ep_h ep, unsigned ep_init_flags,
         return UCS_OK;
     }
 
-    /*
-     * create a wireup endpoint which will start connection establishment
-     * protocol using an auxiliary transport.
-     */
-    if (wiface->attr.cap.flags & UCT_IFACE_FLAG_CONNECT_TO_EP) {
+    return UCS_ERR_UNREACHABLE;
+}
 
-        /* For now, p2p transports have no reason to have proxy */
-        ucs_assert_always(proxy_lane == UCP_NULL_LANE);
+static ucs_status_t
+ucp_wireup_connect_lane_to_ep(ucp_ep_h ep, unsigned ep_init_flags,
+                              ucp_lane_index_t lane, unsigned path_index,
+                              ucp_rsc_index_t rsc_index,
+                              ucp_worker_iface_t *wiface,
+                              const ucp_unpacked_address_t *remote_address)
+{
+    int connect_aux;
+    uct_ep_h uct_ep;
+    ucs_status_t status;
 
-        if (ep->uct_eps[lane] == NULL) {
-            status = ucp_wireup_ep_create(ep, &uct_ep);
-            if (status != UCS_OK) {
-                /* coverity[leaked_storage] */
-                return status;
-            }
+    /* For now, p2p transports have no reason to have proxy */
```

Comment:
not just "for now" - p2p transports do not need proxy because the progress is enabled on both sides